app/components/buildout_design_system/table/table_cell.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module BuildoutDesignSystem
+  class Table
+    class TableCell < ViewComponent::Base
+      attr_reader :class_name
```

Comment:
Maybe there's a parameter we can pass like a boolean for when it's a "header" cell, so it renders a `th` rather than a `td`